dogstatsd/src/aggregator.rs

```diff
@@ -19,6 +19,7 @@ use ustr::Ustr;
 impl MetricValue {
     fn aggregate(&mut self, metric: Metric) {
         // safe because we know there's at least one value when we parse
+        // TODO aggregating different types should return error
```

Comment:
No, it's an implementation details. It's already guarded in the code but this is could be a better place. We are not tracking in Jira all minor improvements to avoid being slowed down